### DISCUSSION TITLE
Shared Document Uneditable

### DISCUSSION THREAD

**Discussion by dwyn** [davidmyersdev/octo#192] [General]
From what I understand, when opening a shared document, the text in the document should almost behave like placeholder text, until a user attempts to edit the shared document.  When the user starts editing, the document then becomes theirs and the url changes from `octo.app/public` to `octo.app/docs/document.id` and the user now has a copy.

I had a friend share a link with me and when I went to edit it, it didn't allow me to change text with my keyboard. I can however manipulate text with the toolbar (changing selected text to bold, italicized, etc). Also, it does not create a copy for my account. 

Any feedback would be greatly appreciated!

**Comment by davidmyersdev** [comment]
Hey @dwyn! I appreciate you bringing this up. The behavior where a doc would be created in your account when editing a shared doc was actually a bug that was fixed [a couple of months ago](https://github.com/writewithocto/octo/commit/63809465faadc9ab780b57737ca83202d0403753). The toolbar, however, is not supposed to allow you to edit the shared doc, so that sounds like a new bug to me.

I definitely get why you'd want the ability to create a copy of a shared doc though, and I think a better UX would be exposing a button where you can "fork" the doc for editing. I'll eventually be adding the ability to collaborate on docs in real-time, so I think the behavior of editing a shared doc should be determined with that in mind.

All that said, I'd love to hear your thoughts!
